### ISSUE TITLE
New section called "Relationship to ARIA"

### ISSUE THREAD

**Issue by LaZay** [mattgarrish/epub-revision#2405]
It would be nice to add a new section called _Relationship to ARIA._ As a matter of fact, there is a lot of misinformation on this topic. 

This new section would be parallel to [Relationship to WCAG](https://www.w3.org/TR/epub-a11y-11/#sec-rel-wcag
), with a very similar (low investment) content, as the philosophy of complementary accessibillity standards is exactly the same in both cases.
 
It goes better when you say it explicitly. Some people do not want to make the investment into ARIA. They pretend that no ARIA at all is necessary for AT in EPUBs, as the EPUB accessibility [v1.1 ](https://www.w3.org/TR/epub-a11y-11/) standard does not even refer to ARIA anymore, when [v1.0](https://www.w3.org/Submission/epub-a11y/) did. Unfortunately, this can be false in some specific use cases.

**Comment by mattgarrish** [comment]
The only reference to ARIA in the 1.0 specification was in the description of optional accessibilityAPI property. That property is no longer referenced, and the use of ARIA as an API was also incorrect and has been fixed in the vocabulary -- it is now a feature.

We're not modifying the requirements on the use of names, roles, states, and properties that go into meeting WCAG. If you don't meet those requirements, you can't claim accessibility. That would include meeting aspects of 1.3.1, 4.1.2, 4.1.3 and probably others that aren't immediately coming to mind.

A separate relationship section can't really be justified in the main specification, in other words, as there's nothing specific to tie it to in the rest of the document. If we want to mention ARIA just for emphasis, I'd replace "rich semantics" in the first sentence of the WCAG relationship section with:

> WCAG [[wcag2](https://www.w3.org/TR/epub-a11y-11/#bib-wcag2)] and its [associated techniques](https://www.w3.org/WAI/WCAG21/Techniques/) provide extensive coverage of issues and solutions for web content accessibility — from tables to embedded multimedia to **_using [[wai-aria]] for interactive content._**

We already mention ARIA often in the [techniques document](https://www.w3.org/TR/epub-a11y-tech-11/), which is where it's more apt since ARIA isn't a conformance standard. If there's something specific about its use in publications that needs explaining, that would be the place to focus on.

**Comment by LaZay** [comment]
You are right. I mixed up the two documents to be honest.
Your proposal is a good idea for the functional document. It should be enough, without a new specific section.
But remember that ARIA is not directly mentionned in the Success Criterion (which addesses only functional user Criterions), Some decision-maker or even digital PM never read the tech part of the WCAG. Sic. They are obsessed with costs and delay. Having to take ARIA into account can be a reality they prefer to deny.

 What about "associated techniques (such as ARIA)" to really underline this other standard than WCAG, and give a point to it?
